modules/README.md

```diff
@@ -4,20 +4,52 @@ Modules should be entirely self-contained into a subdirectory with the following
 
     ModuleName/
 
-        ajax/         - This directory contains any AJAX helper script used by the module. They should
-                        be invoked by using the AjaxHelper.php script in Loris
+        README.md     - This file should contain technical specifications for any module in LORIS.
+                        See any module (ie. modules/candidate_list/README.md) for a template.
```

Comment:
I would have preferred a moc template that to refer to 'any'.  Do we have a guarantee that 'all' README.md file in 'all' module conform?

Reply:
If they don't they should be updated